### ISSUE TITLE
SoftMaxloss vs OIMLoss

### ISSUE THREAD

**Issue by islamamirul** [ShuangLI59/person_search#88]
I am just wondering if you guys have implemented the OIMLoss in caffe since you have reported numbers in your paper using OIMLoss. I trained the network with the provided train.prototxt and got approximately 12% less number than the reported one. I believe the reasoning behind is that the cross entropy loss in the train.prototxt. Could you guys please suggest me ways that can help me to achieve similar numbers you reported in the paper?

**Comment by Cysu** [comment]
In caffe, we use [multiple layers](https://github.com/ShuangLI59/person_search/blob/master/models/psdb/resnet50/train.prototxt#L3077-L3159) together to implement the OIM loss.

Did you change any code / hyperparameters we released in this repo?
